src/api.rs

```diff
@@ -94,7 +94,7 @@ impl<L: Language> From<SyntaxToken<L>> for SyntaxElement<L> {
     }
 }
 
-impl<L: Language> SyntaxNode<L> {
+impl<'a, L: Language> SyntaxNode<L> {
```

Comment:
Shouldn't this `'a` be introduced at `pub fn children_with_tokens_matching`? I'm surprised that an unused lifetime like this is allowed.

Reply:
It should. Fixed!